apiv2/views.py

```diff
@@ -388,6 +388,9 @@ def post(self, request,  *args, **kwargs):
 # SOUND VIEWS
 #############
 
+def get_needs_analyzers_output(fields):
+    return 'analyzers_output' in fields or 'ac_analysis' in fields or '*' in fields
```

Comment:
this logic was used in many places but missed `'*'` from the checks. Move it to a helper function